packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx

```diff
@@ -12,11 +15,12 @@ import {
   getCounterCurrency,
   getOrderType
 } from 'data/components/simpleBuy/model'
+import styled from 'styled-components'
+
+import { BuyOrSell, displayFiat, getOrderDestination } from '../model'
+import { capitalize } from '../../../../utils/capitalize'
```

Comment:
we have an alias for this import.  can just do `import { capitalize } from 'utils/capitalize'`. however lets move this function to `services/validationHelper` so your import will be `import { capitalizeFirstLetter } from 'services/validationHelper'